package.json

```diff
@@ -64,6 +63,7 @@
     "jest": "^24.9.0",
     "nps": "^5.9.8",
     "opt-cli": "^1.6.0",
+    "prettier-eslint": "^9.0.0",
```

Comment:
@lewisl9029 Is this correct?

Reply:
Yep, wanted to make sure this repo itself can dogfood the extension, which requires prettier-eslint to exist in devDependencies.